ssiva13/Questioner
# Ft admin login 163008545

**What Does This PR Do?**
This PR contains the UI template for the 
admin login feature.
**Description of Feature/Task**
Created admin login feature feature.
**_How to Manually test this feature?_**

- Clone the repository using 
- cd into project folder
- Open admin.html using a browser

> Pivotal tracker story ID _#163008545_

The login form feature(as it should be)

![screenshot_2019-01-06 questioner 5](https://user-images.githubusercontent.com/20058478/50740456-bc993280-11ff-11e9-89f2-4bfaac226e0b.png)
